.github/workflows/stark_hash_python.yml

```diff
@@ -0,0 +1,112 @@
+# This file is autogenerated by maturin v0.14.15
+# To update, run
+#
+#    maturin generate-ci -m crates/stark_hash_python/Cargo.toml github
+#
+name: "Python wrapper for stark_hash and stark_poseidon"
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target: [x86_64, aarch64]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
will this then only work with Python 3.10? Wouldn't that be incompatible with the [cairo-lang module](https://pypi.org/project/cairo-lang/)?

Reply:
Ok, great! Could you just, for the sake of my education, point me to the place in the configuration which specifies the various Python versions?

Reply:
Thanks, this alone was useful!